Jotify/Model/Firebase/ReferralManager.swift

```diff
@@ -6,45 +6,26 @@
 //
 
 import FirebaseAuth
-import FirebaseDynamicLinks
 import FirebaseFirestore
+import Airbridge
 
 class ReferralManager {
-    
-    //should be created when a user's account is created
     func createReferralLink() {
         let uid = AuthManager().uid
-        let link = URL(string: "https://jotifyapp.com/?invitedby=\(uid)")
-        let referralLink = DynamicLinkComponents(link: link!, domainURIPrefix: "https://jotify.page.link")
+        let referralLink = "jotify://referral?invitedby=\(uid)"
         
-        referralLink?.iOSParameters = DynamicLinkIOSParameters(bundleID: "com.austinleath.Jotify")
-        referralLink?.iOSParameters?.minimumAppVersion = "2.0.0"
-        referralLink?.iOSParameters?.appStoreID = "1469983730"
+        User.settings?.referralLink = referralLink
         
-        referralLink?.androidParameters = DynamicLinkAndroidParameters(packageName: "com.austinleath.Jotify")
-        referralLink?.androidParameters?.minimumVersion = 1
-                
-        referralLink?.shorten { (shortURL, warnings, error) in
-            if let error = error {
-                print(error.localizedDescription)
-                return
-            }
-            
-            User.settings?.referralLink = shortURL?.absoluteString ?? ""
-            
-            DataManager.updateUserSettings(setting: "referralLink", value: shortURL?.absoluteString ?? "") { success in
-                if !success! {
-                    print("Error creating and uploading referralLink to firestore")
-                }
-                
+        DataManager.updateUserSettings(setting: "referralLink", value: referralLink) { success in
+            if !success! {
+                print("Error creating and uploading referralLink to firestore")
             }
         }
     }
```

Comment:
Refactored referral link creation to use Airbridge's deep link format instead of Firebase Dynamic Links. This change requires coordination with the backend team as it affects how referral tracking works. The new format (jotify://referral?invitedby=<uid>) needs to be properly handled by the deep link processing logic.